config/crd/bases/konk.infoblox.com_konks.yaml

```diff
@@ -35,11 +35,23 @@ spec:
             description: Spec defines the desired state of Konk
             type: object
             x-kubernetes-preserve-unknown-fields: true
+            properties:
+              scope:
+                description: Defines the scope of access to this konk
+                type: string
+                pattern: '^cluster|namespace$'
```

Comment:
is there a reason to ever namespace scope access? Are you thinking a namespace-secure konk? For instance, if you have rbac access to create a certificate, you can use any clusterissuer. It would simplify our test matrix to only think about cluster scoped

Reply:
Yeah, I figure there will be konk users who want it to be namespace-secure. I don't have a user for it now -- our users only want cluster-scoped, and I agree the test matrix is simpler if we only cluster-scope.